pkg/render/node.go

```diff
@@ -1221,11 +1226,25 @@ func (c *nodeComponent) bpffsInitContainer() corev1.Container {
 		Image:           c.nodeImage,
 		ImagePullPolicy: ImagePullPolicy(),
 		Command:         []string{CalicoNodeObjectName, "-init"},
+		Env:             bpffsEnv,
 		SecurityContext: securitycontext.NewRootContext(true),
 		VolumeMounts:    mounts,
 	}
 }
 
+// bpffsEnvVars creates the mount-bpffs container's envvars.
+func (c *nodeComponent) bpffsEnvVars() []corev1.EnvVar {
+	envVars := []corev1.EnvVar{}
+
+	if c.bpfDataplaneEnabled() {
+		if c.cfg.Installation.CustomCgroupPath != "" {
+			envVars = append(envVars, corev1.EnvVar{Name: "CALICO_CGROUP_PATH", Value: c.cfg.Installation.CustomCgroupPath})
```

Comment:
`CustomCgroupPath` doesn't seem like it is a field on the `v1.InstallationSpec` from what I can tell